File: 배열/2563.cpp
#include <iostream>

using namespace std;

int main()
{
	int fld[110][110];
	int T, x, y, sum;

	cin >> T;

	for (int i = 0; i < 110; i++)  //fld 초기화
	{
		for (int j = 0; j < 110; j++)
		{
			fld[i][j] = 0;
		}
	}

	for (int itr = 0; itr < T; itr++)  //
	{
		cin >> x >> y;

		for (int i = x; i < x + 10; i++)
		{
			for (int j = y; j < y + 10; j++)
			{
				fld[i][j] = 1;
			}
		}
	}

	sum = 0;

	for (int i = 0; i < 110; i++)
	{
		for (int j = 0; j < 110; j++)
		{
			sum += fld[i][j];
		}
	}

	cout << sum << endl;

	return 0;
}

File: 배열/2563.java
import java.io.*;

public class Main {
	public static void main(String[] args) throws NumberFormatException, IOException {
		BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
		String str;
		int T = Integer.parseInt(reader.readLine());
		int fld[][] = new int[110][110];
		int x, y, sum = 0;

		for (int i = 0; i < 110; i++)
			for (int j = 0; j < 110; j++)
				fld[i][j] = 0;

		for (int itr = 0; itr < T; itr++) {
			str = reader.readLine();
			x = Integer.parseInt(str.split(" ")[0]);
			y = Integer.parseInt(str.split(" ")[1]);

			for (int i = 0; i < 10; i++)
				for (int j = 0; j < 10; j++)
					fld[y + i][x + j] = 1;
		}

		for (int i = 0; i < 110; i++)
			for (int j = 0; j < 110; j++)
				sum += fld[i][j];

		System.out.println(sum);
	}
}
